Source/missiles.cpp

```diff
@@ -2437,7 +2437,7 @@ void AddFireWallControl(Missile &missile, AddMissileParameter &parameter)
 {
 	std::optional<Point> spreadPosition = FindClosestValidPosition(
 	    [start = missile.position.start](Point target) {
-		    return start != target && IsTileNotSolid(target) && !IsObjectAtPosition(target) && LineClearMissile(start, target);
+		    return start != target && IsTileNotSolid(target) && LineClearMissile(start, target);

```

Comment:
In `GrowWall` we check for `TileProperties::BlockMissile` but `IsTileNotSolid` checks for `TileHasAny(position, TileProperties::Solid)`. Should they both check the same to behave the same?

Reply:
Yes